code/datums/emergency_calls/cryo_marines.dm

```diff
@@ -13,10 +13,12 @@
 	shuttle_id = ""
 	var/leaders = 0
 
-/datum/emergency_call/cryo_squad/spawn_candidates(announce, override_spawn_loc)
+/datum/emergency_call/cryo_squad/spawn_candidates(announce, override_spawn_loc, announce_dispatch_message)
 	var/datum/squad/marine/cryo/cryo_squad = RoleAuthority.squads_by_type[/datum/squad/marine/cryo]
 	leaders = cryo_squad.num_leaders
-	return ..()
+	var/previous_members = members.len
```

Comment:
```suggestion
	var/previous_members = length(members)
```

Reply:
there's no way for members to be null without VVing but ok

Reply:
> there's no way for members to be null without VVing but ok

length() is also faster then .len and i just prefer it for no apparent reason